pkg/pullrequest/api.go

```diff
@@ -152,25 +161,26 @@ func (h *Handler) uploadLabels(ctx context.Context, manifest Manifest, raw []*sc
 			create = append(create, l)
 		}
 	}
-	h.logger.Infof("Creating labels %v for PR %d", create, h.prNum)
+	h.logger.Debugf("Creating labels %v for PR %d", create, h.prNum)
 	for _, l := range create {
-		if _, err := h.client.Issues.AddLabel(ctx, h.repo, h.prNum, l); err != nil {
-			merr = multierror.Append(merr, err)
+		if _, err := h.client.PullRequests.AddLabel(ctx, h.repo, h.prNum, l); err != nil {
+			fmt.Println("ERROR:", err)
```

Comment:
nit: not sure if intentional or not. If intentional, add the name of the method that's failing, else remove.